### DISCUSSION TITLE
What is the minimum requirement of VRAM for this fast repo?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [TheLastBen/fast-stable-diffusion#253] [Q&A] [answered]
Would be nice to know for when local installation be ready...

**Comment by TheLastBen** [comment, marked as answer]
I'll work on it when I have free time

**Reply by toyxyz** [reply]
As always, thank you for your amazing work. I'm looking forward to being able to run it locally too! Colab's free limit is annoying.

**Reply by ZeroCool22** [reply]
> Colab's free limit is annoying.

![this-joker](https://user-images.githubusercontent.com/13344308/198662604-aad1a4b4-28d9-4ec0-a250-82e711bcf242.gif)

**Reply by JasperAlexander** [reply]
Can you tell us when you will have free time? Within days, weeks or months? @TheLastBen

**Reply by TheLastBen** [reply]
https://github.com/TheLastBen/fast-stable-diffusion/blob/main/Local_fast_DreamBooth-Win.ipynb

**Reply by CptGabok** [reply]
I'm super excited this exists, but can someone smarter than me explain how to use this new local version?

I'm just a person on Windows who downloaded the A1111 webui and uses "webui-user.bat" to run it.

**Reply by ZeroCool22** [reply]
> I'm super excited this exists, but can someone smarter than me explain how to use this new local version?
> 
> I'm just a person on Windows who downloaded the A1111 webui and uses "webui-user.bat" to run it.

Meh... couldn't run it, always get some error in the **Colab/Notebook**.

Someone should do a video showing it really working...

**Comment by TheLastBen** [comment]
The local notebook will be a Colab, so you'll need to connect your local runtime to the interface. Because Dreambooth needs the input boxes and the checkboxes to function smoothly. Jupyterlab doesn't offer that feature unfortunately.

**Reply by ZeroCool22** [reply]
> The local notebook will be a Colab, so you'll need to connect your local runtime to the interface. Because Dreambooth needs the input boxes and the checkboxes to function smoothly. Jupyterlab doesn't offer that feature unfortunately.

But be that way, we will still have the time limit, no?

**Reply by TheLastBen** [reply]
Nope, unlimited forever, because you won't be using a google machine
